dirs/dirs.go

```diff
@@ -179,7 +183,8 @@ func SetRootDir(rootdir string) {
 		DistroLibExecDir = filepath.Join(rootdir, "/usr/lib/snapd")
 	}
 
-	CoreLibExecDir = filepath.Join(rootdir, "/usr/lib/snapd")
+	CoreLibExecDir = "/usr/lib/snapd"
+	CoreSnapMountDir = "/snap"
```

Comment:
I suggest keeping the rootdir as originally, and stripping GlobalRootDir out of it when really needed. That makes it consistent.